packages/components/src/components/sheet/sheet.tsx

```diff
@@ -214,7 +214,7 @@ export class Sheet extends LitElement {
   /**
    * When `position` is `"inline-start"` or `"inline-end"`, specifies the width of the component.
    *
-   * @deprecated Use the `width` property instead.
+   * @deprecated in v3.0, removal target v6.0 - Use the `width` property instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use the `width` property instead.
```